apps/ledger-live-desktop/tests/specs/speculos/add.account.spec.ts

```diff
@@ -1,13 +1,12 @@
 import test from "../../fixtures/common";
 import { specs } from "../../utils/speculos";
 import { Currency } from "../../enum/Currency";
-import { Application } from "tests/page";
 import { addTmsLink } from "tests/fixtures/common";
 
 const currencies: Currency[] = [
   Currency.BTC,
   Currency.ETH,
-  Currency.ETC,
+  //Currency.ETC, TODO: enable when bug is merged on develop https://ledgerhq.atlassian.net/browse/LIVE-13049
```

Comment:
To check if fix was merged to develop

Reply:
not fixed, looks like issue is only with speculos